2025-05-29 11:44:24.93
1bb6a832
User
message: 'bind message supplies 1 parameters, but prepared statement "" requires 0',
2025-05-29 11:44:24.93
1bb6a832
User
code: '08P01',
2025-05-29 11:44:24.93
1bb6a832
User
detail: undefined,
2025-05-29 11:44:24.93
1bb6a832
User
severity: 'ERROR',
2025-05-29 11:44:24.93
1bb6a832
User
position: undefined
2025-05-29 11:44:24.93
1bb6a832
User
}
2025-05-29 11:44:24.93
1bb6a832
User
Erro na execução de query após múltiplas tentativas: NeonDbError: bind message supplies 1 parameters, but prepared statement "" requires 0
2025-05-29 11:44:24.93
1bb6a832
User
at execute (file:///home/runner/workspace/node_modules/@neondatabase/serverless/index.mjs:1556:55)
2025-05-29 11:44:24.93
1bb6a832
User
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2025-05-29 11:44:24.93
1bb6a832
User
at async file:///home/runner/workspace/dist/index.js:486:28
2025-05-29 11:44:24.93
1bb6a832
User
at async executeWithRetry (file:///home/runner/workspace/dist/index.js:448:26)
2025-05-29 11:44:24.93
1bb6a832
User
at async Function.query (file:///home/runner/workspace/dist/index.js:478:18)
2025-05-29 11:44:24.93
1bb6a832
User
at async NeonPreparedQuery.execute (file:///home/runner/workspace/node_modules/drizzle-orm/neon-serverless/session.js:76:20)
2025-05-29 11:44:24.93
1bb6a832
User
at async file:///home/runner/workspace/dist/index.js:579:20
2025-05-29 11:44:24.93
1bb6a832
User
at async DatabaseStorage.executeWithRetry (file:///home/runner/workspace/dist/index.js:546:28)
2025-05-29 11:44:24.93
1bb6a832
User
at async DatabaseStorage.getUser (file:///home/runner/workspace/dist/index.js:578:31)
2025-05-29 11:44:24.93
1bb6a832
User
at async file:///home/runner/workspace/dist/index.js:4067:16 {
2025-05-29 11:44:24.93
1bb6a832
User
severity: 'ERROR',
2025-05-29 11:44:24.93
1bb6a832
User
code: '08P01',
2025-05-29 11:44:24.93
1bb6a832
User
detail: undefined,
2025-05-29 11:44:24.93
1bb6a832
User
hint: undefined,
2025-05-29 11:44:24.93
1bb6a832
User
position: undefined,
2025-05-29 11:44:24.93
1bb6a832
User
internalPosition: undefined,
2025-05-29 11:44:24.93
1bb6a832
User
internalQuery: undefined,
2025-05-29 11:44:24.93
1bb6a832
User
where: undefined,
2025-05-29 11:44:24.93
1bb6a832
User
schema: undefined,
2025-05-29 11:44:24.93
1bb6a832
User
table: undefined,
2025-05-29 11:44:24.93
1bb6a832
User
column: undefined,
2025-05-29 11:44:24.93
1bb6a832
User
dataType: undefined,
2025-05-29 11:44:24.93
1bb6a832
User
constraint: undefined,
2025-05-29 11:44:24.93
1bb6a832
User
file: 'postgres.c',
2025-05-29 11:44:24.93
1bb6a832
User
line: '1735',
2025-05-29 11:44:24.93
1bb6a832
User
routine: 'exec_bind_message',
2025-05-29 11:44:24.93
1bb6a832
User
sourceError: undefined
2025-05-29 11:44:24.93
1bb6a832
User
}
2025-05-29 11:44:24.93
1bb6a832
User
Erro na tentativa 1/5: {
